### PR TITLE
added newer pitr & tiflash adapt information

### PR DESCRIPTION
**Pull request by YuJuncen**
Signed-off-by: Yu Juncen <Yu745514916@live.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This PR added the new adaption for tiflash of PiTR.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

N/A

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/pitr-troubleshoot.md [hunk 1]
```diff
@@ -11,12 +11,20 @@ summary: 了解日志备份常见故障以及解决方法。
 
 ## 在使用 `br restore point` 命令恢复下游集群后，无法从 TiFlash 引擎中查询到数据，该如何处理？
 
-在 v6.2.0 版本中，使用 PITR 功能恢复下游集群数据时，并不支持恢复下游的 TiFlash 副本。恢复数据之后，需要执行如下命令手动设置 schema 或 table 的 TiFlash 副本：
+在 v6.2.0 版本中，使用 PiTR 功能恢复下游集群数据时，并不支持恢复下游的 TiFlash 副本。恢复数据之后，需要执行如下命令手动设置 schema 或 table 的 TiFlash 副本：
```
Comment (shichun-0415) [inline br/pitr-troubleshoot.md:14]: 6.2 我们就约定了使用 PITR 了，这是当时统一替换的 PR：[br: replace PiTR with PITR](https://github.com/pingcap/docs-cn/pull/10841)

Comment (shichun-0415) [inline br/pitr-troubleshoot.md:14]:
```suggestion
在 v6.2.0 版本中，使用 PITR 功能恢复下游集群数据时，并不支持恢复下游的 TiFlash 副本。恢复数据之后，需要执行如下命令手动设置 schema 或 table 的 TiFlash 副本：
```

File: br/pitr-troubleshoot.md [hunk 2]
```diff
@@ -11,12 +11,20 @@ summary: 了解日志备份常见故障以及解决方法。
...
 
 ``` shell
 ALTER TABLE table_name SET TIFLASH REPLICA count;
 ```
 
+在 v6.3.0 版本及以上，PiTR 会在恢复完成之后自动依照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` DDL。此时你可以通过以下 SQL 检查 TiFlash 副本的设置状态：
```
Comment (shichun-0415) [inline br/pitr-troubleshoot.md:20]: 目前执行 `ALTER TABLE SET TIFLASH REPLICA` DDL 的主体不太明确，感觉不是 PITR 去执行。
```suggestion
在 v6.3.0 版本及以上，PITR 恢复完成后，BR 会自动按照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` DDL。你可以通过以下 SQL 语句检查 TiFlash 副本的设置状态：
```

File: br/pitr-troubleshoot.md [hunk 3]
```diff
@@ -11,12 +11,20 @@ summary: 了解日志备份常见故障以及解决方法。
...
+
+``` sql
+SELECT * FROM INFORMATION_SCHEMA.tiflash_replica;
+```
+
+需要注意的是，这并不意味着在 PiTR 恢复之后 TiFlash 副本马上可用：在恢复完成之后，TiFlash 需要一定的时间从 TiKV 节点同步数据；PiTR 目前不支持在恢复阶段直接将数据灌入 TiFlash 中。同步进度可以从 `INFORMATION_SCHEMA.tiflash_replica` 表中的 `progress` 一项中看到。
```
Comment (shichun-0415) [inline br/pitr-troubleshoot.md:26]: 这两句中间缺一个衔接信息，也就是什么时候可以同步。
> PITR 目前不支持在恢复阶段直接将数据同步至 TiFlash。同步进度可以从 `INFORMATION_SCHEMA.tiflash_replica` 表中的 `progress` 一项中看到。

另外，原文的 "灌入" 指写入还是指同步呢？
```suggestion
需要注意的是，这并不意味着在 PITR 恢复之后 TiFlash 副本马上可用，TiFlash 需要一定的时间从 TiKV 节点同步数据。因此，PITR 目前不支持在恢复阶段直接将数据同步至 TiFlash。同步进度可以从 `INFORMATION_SCHEMA.tiflash_replica` 表中的 `progress` 一项中看到。
```

File: br/pitr-troubleshoot.md [hunk 4]
```diff
@@ -13,10 +13,18 @@ summary: 了解日志备份常见故障以及解决方法。
 
 在 v6.2.0 版本中，使用 PITR 功能恢复下游集群数据时，并不支持恢复下游的 TiFlash 副本。恢复数据之后，需要执行如下命令手动设置 schema 或 table 的 TiFlash 副本：
 
-``` shell
-ALTER TABLE table_name SET TIFLASH REPLICA count;
+``` sql
+ALTER TABLE table_name SET TIFLASH REPLICA @count;
 ```
 
+在 v6.3.0 版本及以上，PITR 会在恢复完成之后，BR 会自动依照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` DDL。你可以通过以下 SQL 检查 TiFlash 副本的设置状态：
```
Comment (shichun-0415) [inline br/pitr-troubleshoot.md:20]:
```suggestion
在 v6.3.0 版本及以上，PITR 恢复完成之后，BR 会自动依照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` DDL。你可以通过以下 SQL 语句检查 TiFlash 副本的设置状态：
```

File: br/pitr-troubleshoot.md [hunk 5]
```diff
@@ -13,10 +13,18 @@ summary: 了解日志备份常见故障以及解决方法。
...
+
+``` sql
+SELECT * FROM INFORMATION_SCHEMA.tiflash_replica;
+```
+
+需要注意的是，这并不意味着在 PITR 恢复完成之后 TiFlash 副本马上可用，TiFlash 需要一定的时间从 TiKV 节点同步数据，这是因为 PITR 目前不支持在恢复阶段直接将数据写入 TiFlash 中。同步进度可以从 `INFORMATION_SCHEMA.tiflash_replica` 表中的 `progress` 一项中看到。
```
Comment (shichun-0415) [inline br/pitr-troubleshoot.md:26]:
```suggestion
需要注意，PITR 目前不支持在恢复阶段直接将数据写入 TiFlash ，因此 TiFlash 副本在 PITR 完成之后并不能马上可用，而是需要等待一定时间从 TiKV 节点同步数据。要查看同步进度，可以执行 `INFORMATION_SCHEMA.tiflash_replica` 表，查看 `progress` 信息。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Comment by IANTHEREAL** [comment]
@YuJuncen  需要一同修改 1） pitr overview 里面的限制；2） PiTR 的 AP 能力恢复的 RTO 会增加的说明

File: br/point-in-time-recovery.md [hunk 6]
```diff
@@ -86,7 +86,7 @@ PITR 可用于满足以下业务需求：
 - 存储支持 AWS S3 和共享的文件系统（如 NFS 等），暂不支持使用 GCS 和 Azure Blob Storage 作为备份存储。
 - 仅支持集群粒度的 PITR，不支持对单个 database 或 table 执行 PITR。
 - 不支持恢复用户表和权限表的数据。
-- 如果备份集群包含 TiFlash，执行 PITR 后恢复集群的数据不包含 TiFlash 副本，需要手动恢复 TiFlash 副本。具体操作参考[手动设置 schema 或 table 的 TiFlash 副本](/br/pitr-troubleshoot.md#在使用-br-restore-point-命令恢复下游集群后无法从-tiflash-引擎中查询到数据该如何处理)。
+- 如果备份集群包含 TiFlash，执行 PITR 后恢复集群的数据不包含 TiFlash 副本。TiFlash 副本在恢复完成之后会自动进行同步，但是需要消耗一定时间，TiFlash 不一定在恢复完成之后立即可用。
```
Comment (shichun-0415) [inline br/point-in-time-recovery.md:89]:
```suggestion
- 如果备份集群包含 TiFlash，执行 PITR 后恢复集群的数据不包含 TiFlash 副本，而是需要在恢复完成之后从 TiKV 同步数据，这会消耗一定时间。因此，恢复完成之后，TiFlash 不会立即可用。
```

File: br/pitr-troubleshoot.md [hunk 7]
```diff
@@ -13,10 +13,18 @@ summary: 了解日志备份常见故障以及解决方法。
 
 在 v6.2.0 版本中，使用 PITR 功能恢复下游集群数据时，并不支持恢复下游的 TiFlash 副本。恢复数据之后，需要执行如下命令手动设置 schema 或 table 的 TiFlash 副本：
 
-``` shell
-ALTER TABLE table_name SET TIFLASH REPLICA count;
+``` sql
+ALTER TABLE table_name SET TIFLASH REPLICA @count;
 ```
 
+在 v6.3.0 版本及以上，PITR 恢复完成之后，BR 会自动依照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` DDL。你可以通过以下 SQL 语句检查 TiFlash 副本的设置状态：
```
Comment (TomShawn) [inline br/pitr-troubleshoot.md:20]:
```suggestion
在 v6.3.0 及以上版本，PITR 恢复数据完成之后，BR 会自动依照上游对应时刻的 TiFlash 副本数量执行 `ALTER TABLE SET TIFLASH REPLICA` 语句。你可以通过以下 SQL 语句检查 TiFlash 副本的设置状态：
```

File: br/pitr-troubleshoot.md [hunk 8]
```diff
@@ -13,10 +13,18 @@ summary: 了解日志备份常见故障以及解决方法。
...
+
+``` sql
+SELECT * FROM INFORMATION_SCHEMA.tiflash_replica;
+```
+
+需要注意，PITR 目前不支持在恢复阶段直接将数据写入 TiFlash ，因此 TiFlash 副本在 PITR 完成之后并不能马上可用，而是需要等待一定时间从 TiKV 节点同步数据。要查看同步进度，可以查询 `INFORMATION_SCHEMA.tiflash_replica` 表，查看 `progress` 信息。
```
Comment (TomShawn) [inline br/pitr-troubleshoot.md:26]:
```suggestion
需要注意，PITR 目前不支持在恢复阶段直接将数据写入 TiFlash ，因此 TiFlash 副本在 PITR 完成恢复之后并不能马上可用，而是需要等待一些时间从 TiKV 节点同步数据。要查看同步进度，可以查询 `INFORMATION_SCHEMA.tiflash_replica` 表，查看 `progress` 信息。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**